src/main/resources/db/changelog/db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet id="create-project-structure" author="Yankes">
+
+        <createTable tableName="hackathon">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="pk_hackathon_id"/>
+            </column>
+            <column name="description" type="varchar" >
+                <constraints nullable="false"/>
+            </column>
+            <column name="organizer_info" type="varchar" >
+                <constraints nullable="false"/>
+            </column>
+            <column name="event_start_date" type="datetime" >
+                <constraints nullable="false"/>
+            </column>
+            <column name="event_end_date" type="datetime" >
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="team">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="pk_team_id"/>
+            </column>
+            <column type="bigint" name="hackathon_id"/>
+            <column type="bigint" name="owner_id"/>
+        </createTable>
+
+        <createTable tableName="team_tags">
+            <column name="team_id" type="bigint"/>
+            <column name="tag_id" type="bigint"/>
```

Comment:
skoro tagi będą i tak predefiniowane to nie ma sensu po nie uderzać do innego serwisu więc dodaj też tu encję tag proszę bo to raczej się nie będzie za bardzo zmieniać